.circleci/config.yml

```diff
@@ -375,7 +403,6 @@ jobs:
                 include_visit_job_action: true
             - checkout
             - make-deps
-            - run: pip3 install awscli
```

Comment:
I believe this was needed for making api-docs. There was a reason I had to add this in

Reply:
Same comment as above. We may want to stream-line these.